src/encoder.rs

```diff
@@ -67,6 +72,45 @@ impl<W: Write> Encoder<W> {
         Encoder { w, info }
     }
 
+    /// Sets the default frame rate of the animation
+    ///
+    /// It will fail if `set_animation_info` isn't called before
+    pub fn set_frame_rate(&mut self, delay_num: u16, delay_den: u16) -> Result<()> {
```

Comment:
Additionally, why would it be possible to set this aspect specifically but not other parts of `FrameControl`? Special casing frame rate seems appropriate for a format-agnostic interface but not for a mid-level interface as `png` has.

Reply:
Added some check-boxes to the first comment, I forgot about those when I wrote that